client/validate.go

```diff
@@ -55,11 +57,13 @@ func newValidator(params validatorParams) *validator {
 		functionOutputFile:  params.outputFile,
 		stdoutFile:          defaultStdoutFile,
 		stderrFile:          defaultStderrFile,
+		envs: params.envs,
```

Comment:
Is this variable used by the validator?

Reply:
Good catch. I will remove this. It's a leftover variable from a preliminary logic.